formula-test/src/main/java/com/instacart/formula/test/TestFormula.kt

```diff
@@ -106,4 +106,11 @@ abstract class TestFormula<Input, Output> :
         }
         instance.input.interact()
     }
+
+    fun assertRunningCount(expected: Int) {
+        val count = stateMap.size
+        if (count != expected) {
+            throw AssertionError("Expected $expected running formulas, but there were $count instead")
```

Comment:
Just a (possible) clarification point, but it might be worth specifying the formula type in this error message (e.g., "Expected $expected ${instance.javaClass.simpleName} instances running, ..."). That way when the error shows up, it is more immediately obvious which one to look for.

Reply:
Unfortunately, that requires bigger change as `instance.javaclass.simpleName` won't point to anything useful